recipes-soletta/soletta/soletta_0.1.bb

```diff
@@ -4,7 +4,7 @@
 
 DESCRIPTION = "Soletta library and modules"
 SECTION = "examples"
-DEPENDS = "glib-2.0 libpcre pkgconfig"
+DEPENDS = "glib-2.0 libpcre pkgconfig python3-jsonschema-native icu curl"
```

Comment:
I didn't test this recently, but maybe we're also using python3-jsonschema in the generated image? I guess @brunobottazzini can comment on that once he returns from vacations.